File: level1/ex18.js
function solution(numbers) {
  let answer = [];
  const n = numbers.length;

  for (let i = 0; i < n; i++) {
    for (let j = i + 1; j < n; j++) {
      answer.push(numbers[i] + numbers[j]);
    }
  }

  // 중복 제거
  answer = [...new Set(answer)];
  // 오름차순 정렬
  answer.sort((a, b) => a - b);

  return answer;
}

File: level1/ex18.py
from itertools import combinations

def solution(numbers):
    # 2개 요소로 이루어진 모든 조합 구하기
    comb = list(combinations(numbers, 2))
    # 2개 요소 합 구하기
    answer = list(map(lambda x: x[0] + x[1], comb))
    # 중복 제거 & 오름차순 정렬
    answer = sorted(list(set(answer)))
    return answer